hexrd/rotations.py

```diff
@@ -191,11 +184,11 @@ def misorientation(q1, q2, *args):
             % (q1.shape,)
         )
 
-    if len(args) == 0:
+    if symmetries is None:
         # no symmetries; use identity
         sym = (np.c_[1.0, 0, 0, 0].T, np.c_[1.0, 0, 0, 0].T)
     else:
-        sym = args[0]
+        sym = symmetries
```

Comment:
Can simplify this by not setting `sym = symmetries` but instead just replacing all of the `sym`s with `symmetries` in the function. That would also get rid of the `else` clause.